traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -132,6 +133,11 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		/**
 		 * 4.x API
 		 */
+		// CDN lock
+		{api.Version{4, 0}, http.MethodGet, `cdn_lock/?$`, cdn_lock.Read, auth.PrivLevelAdmin, Authenticated, nil, 4134390561},
```

Comment:
i think these should be plural. i think that's the standard. also, probably these query params should be supported:

GET /cdn_locks?cdn=
GET /cdn_locks?username=